app/views/verify/address/index.html.slim

```diff
@@ -1,5 +1,14 @@
-p Pick an address verification mechanism
+h1.h2
+  = t('idv.titles.select_verification')
+p
+  = t('idv.messages.select_verification', sp_name: @sp_name)
```

Comment:
This is the magic ✨  of code review!

The text reads:
> To protect you from identity fraud, you can't use your account at %{sp_name} until you activate it by entering a confirmation code.

Would this make sense with an sp name of `login.gov` ? Maybe! I'd double check with our designer folks.